src/test/TEST_ij/versioncheck.sh

```diff
@@ -19,7 +19,7 @@ if [ -d ../../../.git ]; then
   DEVBRANCH=`git rev-parse --abbrev-ref HEAD`
   if [ -n "$DEVSTRING" ]; then
     if [ "$DEVBRANCH" != "master" ]; then
-      echo "Using HYPRE_DEVELOP_STRING: $DEVSTRING (not main development branch)" \
+      echo "Using HYPRE_DEVELOP_STRING: $DEVSTRING (branch $DEVBRANCH)" \
```

Comment:
Indicate in this message that this is not the main development branch.  You can still print the branch name.

Reply:
Done! Thanks!